jetty-quic/quic-server/src/main/java/org/eclipse/jetty/quic/server/H3Server.puml

```diff
@@ -0,0 +1,65 @@
+@startuml
+title "H3 Quic Server"
```

Comment:
@sbordet @lachlan-roberts @lorban I know it is early days, but I do think puml diagrams like this will help us work out details.
I'm happy to mostly maintain it, but if you can review and comment.